spec/facades/foods_facade_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe FoodsFacade do
```

Comment:
Nice test! Sometimes these tests can feel ridiculously simple but because we are also unit testing our PORO and service, we don't need to make more assertions here because they are making sure we're getting the right JSON structure back and populating the POROs correctly